.github/workflows/linting.yaml

```diff
@@ -0,0 +1,60 @@
+name: code-style
+
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: master
+
+jobs:
+
+  lint-yapf:
+    name: yapf-formatter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install yapf
+        run: |
+          python -m pip install yapf
+      - name: yapf Code Formatter
+        run: |
+          yapf --diff --recursive --parallel .
```

Comment:
Very nice! Since you have this linter I don't think we also need `flake8`. What do you think would be the advantage of running both? The downside is that flake8 may disagree with `yapf` `google` at times which would be frustrating.

Reply:
> What do you think would be the advantage of running both? 

Thinking more about this, I don't see any advantage. So, I removed flake8